### ISSUE TITLE
Variable runtime package name

### ISSUE THREAD

**Issue by joonis** [dashingsoft/pyarmor#1027]
The runtime package/module name is fixed to `pyarmor_runtime_xxxxxx.pyarmor_runtime`.

Would be great if this could be variable. At least to remove the part pyarmor: `runtime_xxxxxx.runtime`

**Comment by jondy** [comment]
It's used to avoid name conflicts.

For example, John publish his obfuscated package. 
Tom obfuscate his script and need import John's obfuscated package.

If the runtime package name is same, only one user's scripts work.

**Comment by joonis** [comment]
Without the `pyarmor_` part it would be unique as well.
And if the runtime files are located inside the package it doesn't matter anyway.

**Comment by jondy** [comment]
I think `pyarmor_runtime_xxxxxx` is more clear, :)

**Comment by joonis** [comment]
What if you make it variable in the pro version?

**Comment by jondy** [comment]
No, the runtime package name could not be changed even in pro/group version.
